python/tests/transformers/tf_tensor_test.py

```diff
@@ -0,0 +1,323 @@
+        # Setup dataset
+        vec_size = 17
+        num_vecs = 31
+        df = self._get_rand_vec_df(num_vecs, vec_size)
+        analyzed_df = tfs.analyze(df)
+        input_col = 'vec'
+        output_col = 'outputCol'
+
+        # Setup saved model export directory
+        saved_model_root = tempfile.mkdtemp()
+        saved_model_dir = os.path.join(saved_model_root, 'saved_model')
+        serving_tag = "serving_tag"
+        serving_sigdef_key = 'prediction_signature'
+
+        builder = tf.saved_model.builder.SavedModelBuilder(saved_model_dir)
+        with tf.Session(graph=tf.Graph()) as sess:
+            # Model definition: begin
+            x = tf.placeholder(tf.float64, shape=[None, vec_size], name='tnsrIn')
+            #x = tf.placeholder(tf.float64, shape=[None, vec_size], name=input_col)
+            w = tf.Variable(tf.random_normal([vec_size], dtype=tf.float64),
+                            dtype=tf.float64, name='varW')
+            z = tf.reduce_mean(x * w, axis=1, name='tnsrOut')
+            # Model definition ends
+
+            sess.run(w.initializer)
+
+            sig_inputs = {
+                'input_sig': tf.saved_model.utils.build_tensor_info(x)}
+            sig_outputs = {
+                'output_sig': tf.saved_model.utils.build_tensor_info(z)}
+
+            serving_sigdef = tf.saved_model.signature_def_utils.build_signature_def(
+                inputs=sig_inputs,
+                outputs=sig_outputs)
+
+            builder.add_meta_graph_and_variables(sess,
+                                                 [serving_tag],
+                                                 signature_def_map={
+                                                     serving_sigdef_key: serving_sigdef
+                                                 })
+            # Get the reference data
+            _results = []
+            for row in df.collect():
+                arr = np.array(row.vec)[np.newaxis, :]
+                _results.append(sess.run(z, {x: arr}))
+            out_ref = np.hstack(_results)
+
+        # Save the model
+        builder.save()
+
+        # Build the transformer from exported serving model
+        # We are using signaures, thus must provide the keys
+        trans_with_sig = TFTransformer(
+            tfInputGraph=TFInputGraphBuilder.fromSavedModel(
+                saved_model_dir, tag_set=serving_tag, signature_def_key=serving_sigdef_key),
+            inputMapping={
+                input_col: 'input_sig'
+            },
+            outputMapping={
+                'output_sig': output_col
+            })
+
+        # Build the transformer from exported serving model
+        # We are not using signatures, thus must provide tensor/operation names
+        trans_no_sig = TFTransformer(
+            tfInputGraph=TFInputGraphBuilder.fromSavedModel(
+                saved_model_dir, tag_set=serving_tag, signature_def_key=None),
+            inputMapping={
+                input_col: 'tnsrIn'
```

Comment:
from our earlier discussion, i thought this would look like 
```
  inputMapping={
    input_col: 'input_sig',
  }
```
no? If `input_col: 'tnsrIn'` is what's expected, we don't need the helper functions or the sigDefKey logic to get the inputMapping and outputMapping, do we?

Reply:
When the user specifically indicated not to use SignatureDef by setting `signature_def_key=None`, we expect the operation names to be provided.